spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/HeapMemoryHealthIndicator.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016-2016 the original author or authors.
```

Comment:
The beginning year should actually be 2012. We can fix that when we merge, though.